src/pages/osobne-udaje.tsx

```diff
@@ -1,59 +1,70 @@
 import React from 'react'
-import { Form, FormikProps } from 'formik'
+import {Form, FormikProps} from 'formik'
 import Link from 'next/link'
-import { FormWrapper, Input } from '../components/FormComponents'
+import {FormWrapper, Input} from '../components/FormComponents'
 import styles from './osobne-udaje.module.css'
 import {
   PersonalInformationUserInput,
   FormErrors,
 } from '../types/PageUserInputs'
-import { getAutoformByPersonName } from '../lib/api'
-import { ErrorSummary } from '../components/ErrorSummary'
+import {getAutoformByPersonName} from '../lib/api'
+import {ErrorSummary} from '../components/ErrorSummary'
 import {
   AutoCompleteInput,
 } from '../components/AutoCompleteInput'
-import { formatPsc, parseFullName } from '../lib/utils'
-import { Nace } from '../components/Nace'
-import { Page } from '../components/Page'
-import { AutoFormSubject } from '../types/api'
+import {formatPsc, getStreetNumber} from '../lib/utils'
+import {Nace} from '../components/Nace'
+import {Page} from '../components/Page'
+import {AutoFormSubject} from '../types/api'
 
 const formatNace = (economicActivity) => {
-  const { code, name } = economicActivity || {}
+  const {code, name} = economicActivity || {}
   if (code && name) {
     return `${code} - ${name}`
   }
   return ''
 }
 
 const makeHandlePersonAutoform = ({
-  setValues,
-}: FormikProps<PersonalInformationUserInput>) => {
-  return (person: AutoFormSubject) => {
-    const { first, last, title } = parseFullName(person.name)
+                                    setValues,
+                                  }: FormikProps<PersonalInformationUserInput>) => {
+  return (subject: AutoFormSubject) => {
+    const {
+      first_name,
+      last_name,
+      prefixes,
+      postfixes,
+      street,
+      reg_number,
+      building_number,
+      municipality,
+      postal_code,
+      country,
+    } = subject.statutory[0]
```

Comment:
Tu by sme mali pridat check na dlzku ze je to non empty array, rovnako je nutne to fixnut aj v odklade

Reply:
Akým spôsobom by som to mal spraviť?
V podstate tú dĺžku checkujeme pri funkcii setValues ako je na obrázku. Ak tie properties neexistujú v tom prázdnom array, tak tam pridá prázdny string.
![asd](https://user-images.githubusercontent.com/90919483/221411072-e26951f4-f85b-48fd-b472-32320c535b49.png)
Najľahší spôsob ako pridať ten check je asi pridať if statement. že ak subject.statutory.length > 0, tak potom vyberiem tie properties z toho.

Reply:
ano musis skontrolovat ze `subject.statutory.length > 0`